libraries/AP_Compass/AP_Compass_AK8963.cpp

```diff
@@ -170,7 +170,7 @@ bool AP_Compass_AK8963::init()
     /* timer needs to be called every 10ms so set the freq_div to 10 */
     if (!_bus->register_periodic_callback(10000, FUNCTOR_BIND_MEMBER(&AP_Compass_AK8963::_update, void))) {
         // fallback to timer
-        _timesliced = hal.scheduler->register_timer_process(FUNCTOR_BIND_MEMBER(&AP_Compass_AK8963::_update_timer, void), 10);
+        hal.scheduler->register_timer_process(FUNCTOR_BIND_MEMBER(&AP_Compass_AK8963::_update_timer, void));
```

Comment:
Do we still want to do this fallback? I think this is the only place I'm aware where we care about the return value of *register_periodic_callback*.